src/entrypoint.sh

```diff
@@ -6,10 +6,16 @@ set -x
 # shellcheck source=src/sync_common.sh
 source sync_common.sh
 
-[ -z "${GITHUB_TOKEN}" ] && {
-  err "Missing input 'github_token: \${{ secrets.GITHUB_TOKEN }}'.";
-  exit 1;
-};
+if [[ -z "${GITHUB_TOKEN}" && -z "${SOURCE_REPO_GITHUB_TOKEN}" ]]; then
```

Comment:
you are able to decide if you want to use GITHUB_TOKEN or SOURCE_REPO_GITHUB_TOKEN. Also possible to sue both options.
The idea is it to make it also possible to have different GitHub hosts (in theory should also be possible to use GitLab,.. with ssh).
https://github.com/AndreasAugustin/actions-template-sync/issues/278

This is just some preparation.